Source/Core/DolphinQt2/NetPlay/NetPlayDialog.h

```diff
@@ -106,4 +109,5 @@ class NetPlayDialog : public QDialog, public NetPlayUI
   GameListModel* m_game_list_model = nullptr;
   bool m_use_traversal = false;
   bool m_is_copy_button_retry = false;
+  int m_buffer_size;
```

Comment:
This should probably have a default initializer to be consistent with the above variables.